test/mockoptimizer.jl

```diff
@@ -32,6 +60,14 @@ end
     MOIT.lin1vtest(optimizer, config)
     optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer, [1.0, 0.0, 2.0], (MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives) => [[0, 2, 0]], (MOI.VectorAffineFunction{Float64}, MOI.Zeros) => [[-3, -1]])
     MOIT.lin1ftest(optimizer, config)
+    optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer, [-4, -3, 16, 0], (MOI.VectorOfVariables, MOI.Nonnegatives) => [[0]], (MOI.VectorOfVariables, MOI.Nonpositives) => [[0]], (MOI.VectorOfVariables, MOI.Zeros) => [[7]], (MOI.VectorAffineFunction{Float64}, MOI.Zeros) => [[7, 2, -4]])
```

Comment:
This is not very readable, could you break into multiple lines with meaningful alignments?

Reply:
Done :)